src/coreclr/scripts/genEventing.py

```diff
@@ -782,7 +782,10 @@ def updateclreventsfile(write_xplatheader, target_cpp, runtimeFlavor, eventpipe_
             Clrallevents.write('#include <PalRedhawk.h>\n')
             Clrallevents.write('#include "clretwallmain.h"\n')
             Clrallevents.write('#include "clreventpipewriteevents.h"\n')
-            Clrallevents.write('#include "EtwEvents.h"\n\n')
+            Clrallevents.write('#ifdef WIN32\n')
+            Clrallevents.write('#include "ClrEtwAll.h"\n')
```

Comment:
This script should also be updated so that the `EventEnabled*` implementations for nativeaot also call `EventXplatEnabled*` when ETW is supported.

https://github.com/dotnet/runtime/blob/5159573eed309f9bf89908bdea7aa2f59ba10878/src/coreclr/scripts/genEventing.py#L440